be/src/runtime/memory/thread_mem_tracker_mgr.h

```diff
@@ -164,7 +162,7 @@ inline void ThreadMemTrackerMgr::pop_consumer_tracker() {
 
 inline void ThreadMemTrackerMgr::consume(int64_t size, bool large_memory_check) {
     _untracked_mem += size;
-    if (!k_doris_run || !ExecEnv::GetInstance()->initialized()) {
```

Comment:
@xinyiZzz  could we remove this variable???

Reply:
It's OK, `ExecEnv.ready` is a static variable
Previously `ExecEnv::GetInstance()` would create `ExecEnv` object